Add more HTTP status codes (#12601)
mdn/content: files/en-us/web/http/status/226/index.md

---
title: 226 IM Used
slug: Web/HTTP/Status/226
tags:
  - HTTP
  - Status code
  - Used
spec-urls: https://www.rfc-editor.org/rfc/rfc3229.html#section-10.4.1
---

{{HTTPSidebar}}

> The HTTP **`226 IM Used`** response code is used when the server has fulfilled a {{HTTPMethod("GET")}} Request for the resource but the response is a representation of the result of one or more instance manipulations applied by combining with previous or future responses as specific instances.

Reviewer: ```suggestion
The HTTP **`226 IM Used`** response code is used when the server has fulfilled a {{HTTPMethod("GET")}} request for the resource but the response is a representation of the result of one or more instance manipulations applied by combining with previous or future responses as specific instances.
```

> The HTTP **`226 IM Used`** response code is used when the server has fulfilled a {{HTTPMethod("GET")}} Request for the resource but the response is a representation of the result of one or more instance manipulations applied by combining with previous or future responses as specific instances.

Reviewer: I don't know what this means. Is it sufficient?

Reviewer 2: I have revamped this article to give context.

---
title: 226 IM Used
slug: Web/HTTP/Status/226
tags:
  - HTTP
  - Status code
  - Used
spec-urls: https://www.rfc-editor.org/rfc/rfc3229.html#section-10.4.1
---

{{HTTPSidebar}}

> **Note:** Browsers don't support _delta encoding_ with HTTP. This status code is sent back by custom servers used by specific clients.

> In the context of delta encodings, the HTTP **`226 IM Used`** status code is set by the server to indicate that it returns a _delta_ to the {{HTTPMethod("GET")}} request it received.

Reviewer: ```suggestion
In the context of delta encodings, the HTTP **`226 IM Used`** status code is set by the server to indicate that it is returning a _delta_ to the {{HTTPMethod("GET")}} request that it received.
```

> Delta encoding consists in sending back to {{HTTPMethod("GET")}} requests differences (called _deltas_) from a given base document. The client uses the `A-IM:` HTTP header to indicate which difference algorithm to use and the `If-None-Match:` header to hint the server about the last version it got. The server generates a delta, sending it back in an HTTP response with the `226` status code and containing the `IM:` (with the name of the algorithm used) and `Delta-Base:` (with the {{HTTPHeader("ETag")}} matching the base document associated to the delta) HTTP headers.

Reviewer: @teoli2003 The English here is hard to parse. Do you mean that you send a get request and get back a response that you can reconstuct if you know a base document it refers to?

I think you might mean:

```suggestion
With delta encoding a server responds to {{HTTPMethod("GET")}} requests with differences (called _deltas_) relative to a given base document (rather than the current document). The client uses the `A-IM:` HTTP header to indicate which differencing algorithm to use and the `If-None-Match:` header to hint the server about the last version it got. The server generates a delta, sending it back in an HTTP response with the `226` status code and containing the `IM:` (with the name of the algorithm used) and `Delta-Base:` (with the {{HTTPHeader("ETag")}} matching the base document associated to the delta) HTTP headers.
```

Reviewer 2: Yes, exactly.
